src/picknik_ur_gazebo_config/objectives/close_cabinet_door.xml

```diff
@@ -1,47 +0,0 @@
-<?xml version="1.0"?>
-<root BTCPP_format="4" main_tree_to_execute="Close Cabinet Door">
-    <BehaviorTree ID="Close Cabinet Door" _description="Close a cabinet door by pushing along an axis with the end effector">
-        <Control ID="Sequence">
-            <SubTree ID="Re-Zero Force-Torque Sensors" />
-            <Action ID="LoadObjectiveParameters" config_file_name="push_along_axis_config.yaml" parameters="{parameters}" />
-            <SubTree ID="CloseGripper" />
-            <Action ID="SaveCurrentState" saved_robot_state="{initial_robot_state}" />
-            <Control ID="Sequence">
-                <Action ID="InitializeMTCTask" task_id="close_cabinet_door" controller_names="/joint_trajectory_controller" task="{close_cabinet_door_task}"/>
-                <Action ID="SetupMTCCurrentState" task="{close_cabinet_door_task}"/>
-                <Action ID="GetPoseFromUser" parameter_name="close_cabinet_door.target_pose" parameter_value="{target_pose}" />
-                <Action ID="SetupMTCMoveToPose" ik_frame="grasp_link" planning_group_name="manipulator" target_pose="{target_pose}" task="{close_cabinet_door_task}" use_all_planners="false"/>
-                <Action ID="SetupMTCUpdateGroupCollisionRule" name="AllowGripperCollisionWithOctomap" parameters="{parameters}" task="{close_cabinet_door_task}" />
-                <Action ID="SetupMTCMoveAlongFrameAxis" task="{push_along_axis_task}" hand_frame="grasp_link" axis_frame="grasp_link" axis_x="0.0" axis_y="0.0" axis_z="1.0" max_distance="1.5" min_distance="0.05" planning_group_name="manipulator" velocity_scale="0.05" acceleration_scale="0.05" />
-                <Action ID="SetupMTCUpdateGroupCollisionRule" name="ForbidGripperCollisionWithOctomap" parameters="{parameters}" task="{close_cabinet_door_task}" />
-                <Action ID="PlanMTCTask" solution="{full_close_cabinet_door_solution}" task="{close_cabinet_door_task}"/>
-                <Action ID="SplitMTCSolution" solution_in="{full_close_cabinet_door_solution}" index="3" solution_out_1="{move_to_approach_solution}" solution_out_2="{push_solution}" />
-                <Action ID="WaitForUserTrajectoryApproval" solution="{full_close_cabinet_door_solution}"/>
-                <Action ID="ExecuteMTCTask" solution="{move_to_approach_solution}"/>
-                <Action ID="SaveCurrentState" saved_robot_state="{pre_approach_robot_state}" />
-                <Control ID="Parallel" success_count="1" failure_count="1">
-                    <Action ID="ForceExceedsThreshold" wrench_topic_name="/force_torque_sensor_broadcaster/wrench" hand_frame_name="grasp_link" wrench_frame_name="tool0" minimum_consecutive_wrench_values="10" force_threshold="10.0"/>
```

Comment:
This is annoying, but the only reason these duplicates were here in the first places is that the values in `ForceExceedsThreshold` Behavior are actually different between sim and hardware :/

Reply:
Does `close_cabinet_door` even work in gazebo? As we cannot open cabinet doors... So what is the reason?

Reply:
Good point. No need to do anything with this one then, but for push button it still does.